README.md

```diff
@@ -17,23 +17,22 @@ function MyButton () {
 }
 MyButton.prototype = Object.create(Nanocomponent.prototype)
 
-MyButton.prototype._render = function (color) {
-  this._color = color
+MyButton.prototype._render = function () {
```

Comment:
I'm a bit hmmm on the `this.props` stuff - perhaps for `nanocomponent` it makes sense to read it from `this.props`, but think in `microcomponent` we should def pass it as an argument?